pkg/storage/kubernetes/ipam.go

```diff
@@ -100,8 +100,8 @@ func toAllocationMap(reservelist []whereaboutstypes.IPReservation, firstip net.I
 }
 
 // GetIPPool returns a storage.IPPool for the given range
-func (i *KubernetesIPAM) GetIPPool(ctx context.Context, ipRange string) (storage.IPPool, error) {
-	normalized := NormalizeRange(ipRange)
+func (i *KubernetesIPAM) GetIPPool(ctx context.Context, ipRange string, networkName string) (storage.IPPool, error) {
```

Comment:
Please make this func receive as input a new type (struct) featuring both the CIDR and the name.

This should be reused in the e2e tests / unit tests.

Reply:
✅